packages/react/src/components/TextArea/TextArea.stories.js

```diff
@@ -206,10 +211,23 @@ export const withAILabel = (args) => {
   );
 };
 
-withAILabel.argTypes = {
-  ...sharedArgTypes,
-};
+withAILabel.args = { ...sharedArgs, helperText: 'Optional helper text' };
 
-export const Skeleton = () => {
-  return <TextAreaSkeleton />;
+export const Skeleton = (args) => {
+  return <TextAreaSkeleton {...args} />;
+};
+Skeleton.args = {
+  ...sharedArgs,
+  labelText: 'Skeleton Label',
+  helperText: undefined,
+  invalid: false,
+  warn: false,
+  enableCounter: false,
+};
+Skeleton.argTypes = {
+  helperText: { control: false },
+  invalid: { control: false },
+  warn: { control: false },
+  enableCounter: { control: false },
+  maxCount: { control: false },
```

Comment:
```suggestion
Skeleton.parameters = {
  controls: {
    include: ['hideLabel'],
  },
};
```

The only control that will work for Skeleton is `hideLabel`, so we can just replace this with a control exclusion.